src/libsyncengine/jobs/network/getappversionjob.h

```diff
@@ -40,16 +40,15 @@ class GetAppVersionJob : public AbstractNetworkJob {
     private:
         std::string getSpecificUrl() override;
         std::string getContentType(bool &canceled) override;
-        void setQueryParameters(Poco::URI &, bool & /*canceled*/) override { /* no query parameters */
-        }
-        void setData(bool & /*canceled*/) override { /* no body parameters */
-        }
+        void setQueryParameters(Poco::URI &uri, bool &canceled) override;
+        void setData(bool & /*canceled*/) override { /* no body parameters */ }
         bool handleError(std::istream &is, const Poco::URI &uri) override;
 
         [[nodiscard]] DistributionChannel toDistributionChannel(const std::string &val) const;
 
         const Platform _platform{Platform::Unknown};
         const std::string _appId;
+        const std::list<int> _userIdList;
```

Comment:
**Suggestion.** Use `std::vector` because the interface is simpler and accessing its element is usually faster. Plus, this is what we already use for the user list.

Reply:
Agreed! Done in c6700de4a937df1ac5a2c4300b373e5c7e1a31c1